grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -1013,28 +1015,27 @@ class OrderService {
 
     def getProductsInOrders(String[] terms, Location destination, Location vendor) {
         return OrderItem.createCriteria().list {
+            createAlias('product', 'prod', JoinType.LEFT_OUTER_JOIN)
+            createAlias('product.synonyms', 'syn', JoinType.LEFT_OUTER_JOIN)
```

Comment:
out of curosity it'd be worth also to check if here `prod.synonyms` would work

Reply:
Yup, it works